src/style.css

```diff
@@ -2,2 +1,103 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: Roboto, Arial, sans-serif;` is a good practice.